src/multivariate/mvnormal.jl

```diff
@@ -22,6 +22,51 @@
 #
 ###########################################################
 
+"""
+
+The [Multivariate normal distribution](http://en.wikipedia.org/wiki/Multivariate_normal_distribution)
+is a multidimensional generalization of the *normal distribution*. The probability density function of
+a d-dimensional multivariate normal distribution with mean vector ``\\boldsymbol{\\mu}`` and
```

Comment:
For whatever reason, the mu and sigma don't seem to be getting rendered properly when I build the docs locally; they show up as ***mu*** and ***Sigma***.